### PR TITLE
Ability to deploy to Tomcat and Heroku

### PR DESCRIPTION
**Pull request by VolodymyrTymtsiv**
Answer to issue "Set up staging serve".

### CODE REVIEW

**Comment by sergeyradkevich** [comment]
@VolodymyrTymtsiv @ShAndriy What is the reason of using `see-lists-with-items` branch here?

**Comment by sergeyradkevich** [comment]
The name of the pull request should be about support of deploying.

**Comment by sergeyradkevich** [comment]
I suppose a wrong name was taken based on the branch name, please correct.

**Comment by VolodymyrTymtsiv** [comment]
We use this branch because we must did it in first user story that see-lists-with-items.
So,create new branch and change in this pull request or let this time as there.

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# TaDa
+# tada
+To deploy project on heroku you must:
+
+* Install heroko.
+    - sudo add-apt-repository "deb https://cli-assets.heroku.com/branches/stable/apt ./"
```
Comment (sergeyradkevich) [inline README.md]: Correct a typo please: herok*u*

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+    - curl -L https://cli-assets.heroku.com/apt/release.key | sudo apt-key add -
+    - sudo apt-get update
+    - sudo apt-get install heroku.
+* Register on heroku.
```
Comment (sergeyradkevich) [inline README.md]: I suppose `.` is not a part of the command, so remove it please.

> sudo apt-get install heroku

File: pom.xml [hunk 3]
```diff
@@ -16,6 +16,14 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>com.heroku.sdk</groupId>
+                <artifactId>heroku-maven-plugin</artifactId>
+                <version>1.1.2</version>
+                <configuration>
+                    <appName>tada-prod-server</appName>
+                </configuration>
```
Comment (sergeyradkevich) [inline pom.xml]: Guys, move this name into the resources (so it is hard-coded now). It should be a variable instead here that will be set during maven execution automatically.

File: README.md [hunk 4]
```diff
@@ -1 +1,21 @@
...
+* Clone a project with github.
+* Create personal app in heroku
+    - Create app in web interface on heroku
+    - Type "heroku create" in terminal opened on cloned project.
+* Configure pom.xml:
+    - Open pom.xml
+    - In com.heroku.sdk plugin change app name to your created app name.
```
Comment (sergeyradkevich) [inline README.md]: Remove description about pom.xml (as it was suggested to use build properties instead of to be hard-coded).

**Comment by VolodymyrTymtsiv** [comment]
Please check whether all good.

**Comment by sergeyradkevich** [comment]
Will take a look soon. Guys, let's use some phrase like '@<reviewer> Please review' when asking for a review.

**Comment by VolodymyrTymtsiv** [comment]
@sergeyradkevich Ok, tomorrow will do everything as agree.

**Comment by sergeyradkevich** [comment]
Let me remind to use some conventions during our communications here:

> Guys, let's use some phrase like '<@reviewr> Please review' when asking for a review.

Do not use general phrases like "Please check whether all good", instead use phrases that describe the intent/action.

**Comment by ShAndriy** [comment]
@sergeyradkevich Updated, review again please.

**Comment by sergeyradkevich** [comment]
Delete given `see-lists-with-items` branch once it isn`t being used.

**Pull request merged**